arbnode/node.go

```diff
@@ -93,6 +93,7 @@ type Config struct {
 	TransactionStreamer TransactionStreamerConfig   `koanf:"transaction-streamer" reload:"hot"`
 	Maintenance         MaintenanceConfig           `koanf:"maintenance" reload:"hot"`
 	ResourceMgmt        resourcemanager.Config      `koanf:"resource-mgmt" reload:"hot"`
+	SnapSync            SnapSyncConfig              `koanf:"snap-sync"`
```

Comment:
I don't want anyone to actually configure these. They should only be used in tests.
Will that work?
* rename SnapSync to SnapSyncTest or something like this, with a comment explaining generally what it does
* remove the koanf comment of this and any sub-config
* remove SnapSyncAddOptions func

Reply:
done.